models/ampleforth/ethereum/ampleforth_ethereum_airdrop_claims.sql

```diff
@@ -0,0 +1,41 @@
+{{
+    config(
+        alias='airdrop_claims',
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['recipient', 'tx_hash', 'evt_index'],
+        post_hook='{{ expose_spells(\'["ethereum"]\',
+                                "project",
+                                "ampleforth",
+                                \'["hildobby"]\') }}'
+    )
+}}
+
+SELECT 'ethereum' AS blockchain
+, t.evt_block_time AS block_time
+, t.evt_block_number AS block_number
+, 'Ampleforth' AS project
+, 'Ampleforth Forth Airdrop' AS airdrop_identifier
+, t.to AS recipient
+, t.contract_address
+, t.evt_tx_hash AS tx_hash
+, CAST(t.value AS DECIMAL(38,0)) AS amount_raw
+, CAST(t.value/POWER(10, 18) AS double) AS amount_original
+, CAST(pu.price*t.value/POWER(10, 18) AS double) AS amount_usd
+, '0x77fba179c79de5b7653f68b5039af940ada60ce0' AS token_address
+, 'FORTH' AS token_symbol
+, t.evt_index
+FROM {{ source('erc20_ethereum', 'evt_transfer') }} t
+LEFT JOIN {{ ref('prices_usd_forward_fill') }} pu ON pu.blockchain = 'ethereum'
+    AND pu.contract_address='0x77fba179c79de5b7653f68b5039af940ada60ce0'
+    AND pu.minute=date_trunc('minute', t.evt_block_time)
+    {% if is_incremental() %}
+    AND pu.minute >= date_trunc("day", now() - interval '1 week')
+    {% endif %}
+WHERE t.contract_address = '0x77fba179c79de5b7653f68b5039af940ada60ce0'
+AND from = '0xf497b83cfbd31e7ba1ab646f3b50ae0af52d03a1'
+AND t.evt_block_time BETWEEN '2021-04-20' AND '2022-04-16'
```

Comment:
i assume with this filter, that means the airdrop claim is over? in that case, i'd suggest we remove logic for incremental and make it a table. we'd also want to apply a 'static' tag in the config block of the model, that way our prod job doesn't run it each time as there is no new data. please look to apply this logic throughout any spell in this PR that fits that criteria